spacy_llm/tests/tasks/test_spancat.py

```diff
@@ -318,43 +390,43 @@ def test_spancat_alignment(response, alignment_mode, gold_spans):
 def test_invalid_alignment_mode():
     labels = "PER,ORG,LOC"
     with pytest.raises(ValueError, match="Unsupported alignment mode 'invalid"):
-        make_spancat_task_v2(labels=labels, alignment_mode="invalid")
+        make_spancat_task_v3(examples=[], labels=labels, alignment_mode="invalid")
 
 
 @pytest.mark.parametrize(
-    "response,case_sensitive,single_match,gold_spans",
+    "response, case_sensitive, gold_spans",
     [
         (
-            "PER: Jean",
-            False,
-            False,
-            [("jean", "PER"), ("Jean", "PER"), ("Jean", "PER")],
-        ),
-        (
-            "PER: Jean",
+            "1. Jean | True | PER | is a person's name",
             False,
-            True,
             [("jean", "PER")],
         ),
         (
-            "PER: Jean",
-            True,
-            False,
-            [("Jean", "PER"), ("Jean", "PER")],
-        ),
-        (
-            "PER: Jean",
-            True,
+            "1. Jean | True | PER | is a person's name",
             True,
             [("Jean", "PER")],
         ),
+        # TODO: Get this matching working again, issue is with the parsing
```

Comment:
This TestCase doesn't work and it's pretty tough to get it to work properly:

Reply:
Can you elaborate on why it's tough?

Reply:
This is the test case referenced below

Reply:
`This guy jean (or Jean) is the president of the Jean Foundation.`, right? Is it still difficult to make this work with the different parsing you proposed?

Reply:
Yes it is, unless we do the separate step of first case sensitive matching, then case insensitive matching afterward. Explained in more detail below